src/main/java/ch/heigvd/gen/monopoly/MonopolyGame.java

```diff
@@ -42,7 +46,7 @@ public MonopolyGame(String[] players) throws IllegalArgumentException {
      * Méthode qui lance le jeu
      */
     void playGame() {
-        for (; currentRound < NUMB_OF_ROUNDS; ++currentRound) {
+        while (currentRound < NUMB_OF_ROUNDS) {
```

Comment:
Why `currentRound` is incremented in `playRound()` ?

Reply:
My bad ^^ I will correct that